config-default.yaml

```diff
@@ -0,0 +1,60 @@
+#######################################################################################
+##                                  DEFAULT CONFIG                                   ##
+#######################################################################################
+# - Defines the default configuration
+# - **WARNING**: Do not modify this file
+#   - For custom config create a new file "custom/config.yaml"
+#   - You can override in the custom file any config that is defined here
+#   - For redefining components, or for more information, follow the instructions in ./README.md
+
+# TODO: Rename Dapp into Gnosis Protocol DApp
```

Comment:
Ups, this part was not implemented yet: https://github.com/gnosis/dex-react/issues/1045

Reply:
Done!